Source/WebCore/accessibility/AXObjectCache.h

```diff
@@ -543,6 +543,14 @@ class AXObjectCache final : public CanMakeWeakPtr<AXObjectCache>, public CanMake
     Ref<Document> protectedDocument() const;
     constexpr const std::optional<PageIdentifier>& pageID() const { return m_pageID; }
 
+#if !ENABLE(INCLUDE_IGNORED_IN_CORE_AX_TREE) && ENABLE(ACCESSIBILITY_ISOLATED_TREE)
```

Comment:
Should we put a `FIXME` n front of one of these that `objectBecameIgnored` can be removed once `INCLUDE_IGNORED_IN_CORE_AX_TREE` is enabled?

Reply:
IMO that's implicit, as when `INCLUDE_IGNORED_IN_CORE_AX_TREE` is removed and defaulted to be "on", any code that is `!ENABLE(INCLUDE_IGNORED_IN_CORE_AX_TREE)` only has no reason to exist 🤔

Reply:
Fair point, we can leave it out